modules/rosa-planning-environment-cluster-max.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// rosa_planning/rosa-planning-environment.adoc
+
+[id="planning-environment-cluster-maximums_{context}"]
+= Planning your environment based on tested cluster maximums
+
+This document describes how to plan your {product-title} environment based on the tested cluster maximums.
+
+Oversubscribing the physical resources on a node affects resource guarantees the Kubernetes scheduler makes during pod placement. Learn what measures you can take to avoid memory swapping.
+
+Some of the tested maximums are stretched only in a single dimension. They will vary when many objects are running on the cluster.
+
+The numbers noted in this documentation are based on Red Hat’s test methodology, setup, configuration, and tunings. These numbers can vary based on your own individual setup and environments.
```

Comment:
FWIW, I take the IBM stance on not using the possessive form of a company (around p67 in the ISG).  So, maybe "...on Red Hat testing methodology,..."  Up to you.